signalbot/context.py

```diff
@@ -27,6 +27,22 @@ async def send(
             mentions=mentions,
             text_mode=text_mode,
         )
+    async def edit(
+            self, 
+            text: str,
+            base64_attachments: list = None,
+            mentions: list = None,
+            text_mode: str = None,
+    ):
```

Comment:
The code checks are failing in the CI, it looks like you forgot to install `pre-commit` as described here https://github.com/filipre/signalbot?tab=readme-ov-file#local-development Please do, then you can locally run precommit also with `pre-commit run` or `pre-commit run --all` if you've already committed changes.